src/utils/network/session-core.js

```diff
@@ -560,3 +567,76 @@ export const createInbox = async (session, podUrl) => {
     await setDocAclForPublic(session, inboxContainerUrl, { append: true });
   }
 };
+
+/**
+ * A function that shows the documents from Solid Pod from user into PASS
+ *
+ * @function getDocTTLs
+ * @param {Session} session - Solid's Session Object {@link Session}
+ * @param {URL} podUrl - Pod URL of user to show documents from
+ * @returns {Promise<object[]>} - List of parsed objects and/or error status codes
+ */
+
+export const getDocTTLs = async (session, podUrl) => {
```

Comment:
A lot of this duplicate functionality of what's being written in https://github.com/codeforpdx/PASS/pull/254

We probably don't want to be implementing the same functionality in multiple places in the code base.

Reply:
I believe only this function fetches for `{ uploadDate, filename, documentType: docType, expireDate, description, documentUrl }` at this time from `document.ttl`? I do find functions that builds and updates `document.ttl` in session-core and session-helper.

Do you mean to refactor the try portion of the codebase out as a helper function?